notebook-lm-clone/src/podcast/script_generator.py

```
@@ -0,0 +1,299 @@
+    def generate_script_from_website(
+        self,
+        website_chunks: List[Any],
+        source_url: str,
+        podcast_style: str = "conversational",
+        target_duration: str = "10 minutes"
+    ) -> PodcastScript:
+
+        logger.info(f"Generating podcast script from website: {source_url}")
+        
+        if not website_chunks:
+            raise ValueError("No website content provided")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)